cmd/xl-v1-object.go

```diff
@@ -680,8 +680,7 @@ func (xl xlObjects) PutObject(bucket string, object string, size int64, data io.
 	}
 
 	// Rename the successfully written temporary object to final location.
-	onlineDisks, err = renameObject(onlineDisks, minioMetaTmpBucket, tempObj, bucket, object, xl.writeQuorum)
-	if err != nil {
+	if _, err = renameObject(onlineDisks, minioMetaTmpBucket, tempObj, bucket, object, xl.writeQuorum); err != nil {
```

Comment:
If we are not going to use it further its not useful, a function signature already communicates what it does.  Is onlineDisks being used again ? if not we shouldn't set it.  We should review when such a code is added in future to use onlineDisks by using the value set by the renameObject() 

Otherwise it doesn't make sense to set it IMO if we are not going to use it further.